components/style/properties/gecko.mako.rs

```diff
@@ -924,7 +924,7 @@ fn static_assert() {
                     structs::${border_style_keyword.gecko_constant(value)} ${border_style_keyword.maybe_cast("u8")},
             % endfor
                 Either::First(Auto) =>
-                    structs::${border_style_keyword.gecko_constant('solid')} ${border_style_keyword.maybe_cast("u8")},
+                    structs::${border_style_keyword.gecko_constant('auto')} ${border_style_keyword.maybe_cast("u8")},
```

Comment:
@Wafflespeanut @emilio Can you let me know if this is the right approach for usage of gecko based prefs?